sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -456,21 +455,32 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
 
   test("literal in agg grouping expressions") {
     checkAnswer(
-      sql("SELECT a, count(1) FROM testData2 GROUP BY a, 1"),
-      Seq(Row(1, 2), Row(2, 2), Row(3, 2)))
+      sql("SELECT SUM(a) FROM testData2 GROUP BY 2"),
+      Seq(Row(6), Row(6)))
+
     checkAnswer(
-      sql("SELECT a, count(2) FROM testData2 GROUP BY a, 2"),
+      sql("SELECT a, SUM(b) FROM testData2 GROUP BY 1"),
+      Seq(Row(1, 3), Row(2, 3), Row(3, 3)))
+
+    checkAnswer(
+      sql("SELECT a, count(1) FROM testData2 GROUP BY a, 1"),
       Seq(Row(1, 2), Row(2, 2), Row(3, 2)))
 
     checkAnswer(
       sql("SELECT a, 1, sum(b) FROM testData2 GROUP BY a, 1"),
```

Comment:
If we have a table with a single column, what will happen if we call `GROUP BY 1, 2`?